site/data/charts/docker.json

```diff
@@ -31,6 +31,7 @@
     ["2024-05-01", 0, 0, 0, 0, 2193, 350, 65308, 1704, 0, 0, 0, 0, 0, ""],
     ["2024-06-01", 0, 0, 0, 0, 3309, 583, 124015, 2135, 422, 211, 13148, 1849, 184536, ""],
     ["2024-07-01", 0, 0, 0, 0, 3842, 1834, 99453, 3026, 442, 1516, 19004, 3795, 166520, ""],
-    ["2024-08-01", 0, 0, 0, 0, 4271, 1538, 89718, 2696, 432, 2383, 24987, 2141, 186036, ""]
+    ["2024-08-01", 0, 0, 0, 0, 4271, 1538, 89718, 2696, 432, 2383, 24987, 2141, 186036, ""],
+    ["2024-10-01", 0, 0, 0, 0, 3502, 818, 30953, 1288, 479, 842, 32681, 1785, 754399, ""]
```

Comment:
Me September?

Reply:
We effectively lost the data because the actions got disabled.
That means Sept includes 2 months data. I can guestimate the prev month values to make it look more realistic?
We should have data from near the start of the month, so its wont be a complete guess..